pkg/upload/dsym.go

```diff
@@ -187,15 +188,17 @@ func ProcessDsym(
 			err = server.ProcessFileRequest(endpoint+"/dsym", uploadOptions, fileFieldData, timeout, retries, dsym.UUID, dryRun)
 
 			if err != nil {
-				if failOnUpload {
-					return err
-				} else {
-					log.Warn(err.Error())
+				if strings.Contains(err.Error(), "404 Not Found") {
+					log.Info("Trying " + endpoint)
```

Comment:
Should be elaborate for the user what happened here before the retry? cc: @tomlongridge for your opinion too

Reply:
The user doesn't have any control over which path to use, so I'd actually suggest we don't log at all here. But I agree, we should either log both attempts or none. (Consider what we do on other platforms – maybe introducing a log the endpoint used across all platforms is something we raise as a small future task.)